src/legacy/core_plugins/kibana/public/management/sections/index_patterns/edit_index_pattern/components/index_header.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiTitle, EuiButtonEmpty, EuiIcon } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  defaultIndex: boolean;
+  indexPatternTitle: string;
+  setDefault: any;
+  refreshFields: any;
+  removePattern: any;
```

Comment:
They are angular functions and should be re-written later. So, they should be left as `any`.